lib/media/drm_engine.js

```diff
@@ -774,6 +783,48 @@ shaka.media.DrmEngine.prototype.configureClearKey_ = function() {
 };
 
 
+/**
+ * Returns the DrmInfo that is generated by drm configation.
+ * It activates DrmEngine if drm configs have keySystems.
+ * @param {!shakaExtern.Manifest} manifest
+ * @return {(Array.<{object}>|null)}
+ * @private
+ */
+shaka.media.DrmEngine.prototype.getDrmInfosByConfig_ =
+    function(manifest) {
+  var config = this.config_;
+  if (!config) {
+    return null;
+  }
+
+  var serverKeys = Object.keys(config.servers || {});
+
+  if (!serverKeys.length) {
+    return null;
+  }
+
+  var isEncryptedContent = manifest.periods.every(function(period) {
+    period.variants.every(function(variant) {
+      return !!variant.drmInfos.length;
```

Comment:
No need to do !!, since length is a number which will implicitly convert to boolean.